graph/schema.resolvers.go

```diff
@@ -42,6 +43,20 @@ func (r *mutationResolver) NormalizeTest(ctx context.Context, id string) (bool,
 	return true, nil
 }
 
+func (r *mutationResolver) NormalizeTests(ctx context.Context, ids []string) (bool, error) {
```

Comment:
You should also add a test for this call. I'll add you to the keploy account for this project. There you need to just perform this request, and it should get captured in testmode.

Reply:
Have you added me ?

Reply:
sending you the creds on slack :)